packages/react/src/app-layout/AppLayoutFooter.tsx

```diff
@@ -1,16 +1,30 @@
 import { PropsWithChildren } from 'react';
-import { Stack } from '../stack';
+import { Box } from '../box';
 import { Content } from '../content';
-import { boxPalette } from '../core';
+import { boxPalette, type BoxPalette, type ResponsiveProp } from '../core';
+import { Stack } from '../stack';
 
 export type AppLayoutFooterProps = PropsWithChildren<{
+	background?: 'body' | 'bodyAlt';
 	/** Defines an identifier (ID) which must be unique. */
 	id?: string;
+	palette?: ResponsiveProp<BoxPalette>;
```

Comment:
Since we export the `AppLayoutFooter`, we should add JSDocs for the new props `background` and `palette`.

Reply:
Looks like there are more components without `background` props docs than with. I don't mind adding it, but is "/** The background of the component */" like in HeroCategoryBanner actually benefit users?

Reply:
Perhaps it'd be better to define it once and use it from there? Maybe not something we worry about in this PR and we can come back and tidy them all up together? This will have a gap in our props table on our docs site in the meantime, though.